internal/provider/envgroup/resource/link.go

```diff
@@ -63,6 +63,24 @@ func (r *envGroupLinkResource) Create(ctx context.Context, req resource.CreateRe
 		return
 	}
 
+	existingEnvGroup, err := r.getEnvGroup(ctx, plan.EnvGroupId.ValueString())
+	if err != nil {
+		resp.Diagnostics.AddError("unable to get environment group", err.Error())
+		return
+	}
+
+	// Attempting to create a service link for an env group that already exists and is linked to another service
+	// will result in an inconsistent state error. The user should instead import the existing env group link and
+	// update it. We check to see if the service link already exists and contains a service ID not in the plan.
+	for _, id := range existingEnvGroup.ServiceLinks {
+		if !slices.Contains(plan.ServiceIds, id.Id) {
+			resp.Diagnostics.AddError(
+				"already exists",
+				fmt.Sprintf("service link already exists for %s", existingEnvGroup.Id))
```

Comment:
```suggestion
				fmt.Sprintf("service link already exists for %s please import the service link or add linked services to the terraform file", existingEnvGroup.Id))
```

Maybe give the user a little more info on how to fix the issue?

Reply:
Stuck with just suggesting importing, since that encompasses both running the import command and adding it to the Terraform file.